docs/ext/bridge/index.rst

```diff
@@ -0,0 +1,93 @@
+.. _discord_ext_bridge:
+
+``discord.ext.bridge`` -- A module that bridges prefix commands to slash commands
```

Comment:
I think this would be a bit better since the examples use `ctx.respond` which is for application commands, showing that this module is built around that
```suggestion
``discord.ext.bridge`` -- A module that bridges slash commands to prefixed commands
```